### PR TITLE
Correcting Knative Package Failures

### PR DESCRIPTION
**Pull request by hemantsaini-7**
opening a PR for checking nightly failures of knative package, and correcting them.

**Jira Id:**
- Story: [ODC-7224](https://issues.redhat.com/browse/ODC-7224)

### CODE REVIEW

File: test-prow-e2e.sh [hunk 1]
```diff
@@ -81,7 +81,7 @@ if [ "$SCENARIO" == "nightly-cypress" ]; then
   fi
   ./test-cypress.sh -n true $PACKAGE
 elif [ "$SCENARIO" == "e2e" ] || [ "$SCENARIO" == "release" ]; then
-  ./test-cypress.sh -h true
+  ./test-cypress.sh -h true -n true -p knative
```
Comment (invincibleJai) [inline test-prow-e2e.sh:84]: do we need this?

Comment (hemantsaini-7) [reply test-prow-e2e.sh:84]: it's only for testing.

**Comment by hemantsaini-7** [comment]
PR affects e2e tests only

/label docs-approved
/label px-approved
/label qe-approved

**Pull request merged**